plugins/com.google.cloud.tools.eclipse.util/templates/appengine/ObjectifyWebListener.java.ftl

```diff
@@ -12,7 +12,7 @@ public class ObjectifyWebListener implements ServletContextListener {
   @Override
```

Comment:
`@WebListener` above should be removed in case of Servlet 2.5. It leads to a compilation error, as 2.5 API doesn't have the `javax.servlet.annotation` package.

Reply:
done

Reply:
The `WebListener` import should be removed too when Servlet == 2.5.

Reply:
done